packages/spright-components/src/fast-graph/index.ts

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable import/extensions */
+/* eslint-disable import/no-extraneous-dependencies */
+import { observable } from '@ni/fast-element';
+import { DesignSystem, FoundationElement } from '@ni/fast-foundation';
+import 'flot/source/jquery';
```

Comment:
Another thought is if webpack is treating  jquery as an import (from discussion earlier) you may be able to do:
```
import jQuery from 'jquery';
window.jQuery = jQuery;
```